carbonmark/lib/retirementDataQueries/getProjectInfoViaPolygonDigitalCarbon.ts

```diff
@@ -0,0 +1,57 @@
+import { subgraphs } from "@klimadao/lib/constants";
+
+export const getProjectInfoViaPolygonDigitalCarbon = async (
```

Comment:
Hold up.. shouldn't this be handled by the API?

Sorry if you already mentioned this elsewhere.

Reply:
Creating the github issue as discussed